### ISSUE TITLE
Generators seems to be broken

### ISSUE THREAD

**Issue by antonkomarev** [jadjoubran/laravel5-angular-material-starter#183]
Hello, I've runned composer update today and generators gone down.

Right now in 3.0 they are setted as:

``` json
"laravelangular/generators": "^1.4",
```

In case of `composer update` they are upgrading to v1.6.1:

``` sh
Loading composer repositories with package information                                                                                                                Updating dependencies (including require-dev)         
  - Removing laravelangular/generators (1.4.0)
  - Installing laravelangular/generators (1.6.1)
    Loading from cache
```

After update generators start to work with problems because they are not planned for 3.0, am I right?

I've downgraded package version in composer.json manually and that fixed issues:

``` json
"laravelangular/generators": "1.4",
```

**Comment by jadjoubran** [comment]
Hi @a-komarev 

Thanks for reporting this. Yes indeed this is a bug in 3.0 (3.0.6) that has already been fixed in 3.1
I'm going to patch it in 3.0.7

**Comment by jadjoubran** [comment]
The best approach would be to have it as `"~1.4"`

**Comment by jadjoubran** [comment]
Fixed @a-komarev [here](https://github.com/jadjoubran/laravel5-angular-material-starter/commit/c33d8192c04cbe5b32dece934d8a6e326ab0f7a5)
Thank you for your contribution :smile:

**Comment by antonkomarev** [comment]
@jadjoubran no problem! Thanks for your work!

**Comment by antonkomarev** [comment]
@jadjoubran Not sure this helped :( It should be `~1.4.0` or `1.4.*`.

**Comment by jadjoubran** [comment]
Yes you're right
I'll fix it as soon as I'm done with the v3.1 release :smile:

**Comment by jadjoubran** [comment]
Fixed in 3.0.8
Thanks!
